### PR TITLE
Add flower crown

### PR DESCRIPTION
**Pull request by Partmedia**
## About the PR
It's a crown of flowers, an upgrade from the existing 'hairflower'. It can be constructed with a poppy flower.

> Her dotage now I do begin to pity,
> For, meeting her of late behind the wood
> Seeking sweet favors for this hateful fool,
> I did upbraid her and fall out with her,
> For she his hairy temples then had rounded
> With **a coronet of fresh and fragrant flowers**.

From Act 4, Scene 1, *A Midsummer Night's Dream* by William Shakespeare

**Screenshots**

flower-crown

**Changelog**

A little secret; no change log entry.

### CODE REVIEW

File: Resources/Prototypes/Recipes/Crafting/Graphs/improvised/flowercrown.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+- type: constructionGraph
+  id: flowercrown
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: flowercrown
+          steps:
+            - prototype: FoodPoppy
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Recipes/Crafting/Graphs/improvised/flowercrown.yml:9]: For this construction step, do not use the prototype step. You'll want to either create a tag or a material for flowers in general. This also means that you could expand this to work with different flowers. You can read more about construction here: [https://docs.spacestation14.io/en/content/construction/index](url)

Comment (Partmedia) [reply Resources/Prototypes/Recipes/Crafting/Graphs/improvised/flowercrown.yml]: Added "Flower" tag, converted to use Flower.

**Comment by Elijahrane** [comment]
If it's a single ingredient thing why not just make the poppy wearable like ambrosia is?

**Comment by Partmedia** [comment]
> If it's a single ingredient thing why not just make the poppy wearable like ambrosia is?

That would be a better question for whoever introduced hair flowers without a construction recipe.

Visually, the flower crown looks different from a single flower. So, it makes sense that you have to craft it.

If the concern is that the construction recipe isn't ideal, we can improve it. I intended for the recipe to require multiple flowers, but I was only able to find the poppy flower. And, per @EmoGarbage404's suggestion, unless I add a separate tag for each color of flower, the recipe won't be able to distinguish between different flower colors, either. Would it be better to just remove the construction recipe?

Another alternative might be: Poppy --> Hair flower --(add another poppy)--> Flower crown

**Comment by EmoGarbage404** [comment]
I think this current version is pretty good. It looke pretty good to me and can always be expanded with more flowers in the future.

**Review by EmoGarbage404 — APPROVED** [review]

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**